v0.0.1/admin-schema.json

```
@@ -0,0 +1,633 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/main/v0.0.1/admin-schema.json",
+    "title": "Hub administrative settings",
```

```suggestion
    "title": "Modeling Hub administrative settings",
```
Suggestion (will add to other files) to modify all titles to have similar naming convention.